src/main/java/com/google/firebase/messaging/MulticastMessage.java

```diff
@@ -116,10 +118,10 @@ public Builder addToken(@NonNull String token) {
      * Adds a list of tokens to which the message should be sent. Up to 100 tokens can be
```

Comment:
Please update to `Adds a collection tokens to which...`.

Also please specify that the tokens will be added in the iteration order of the collection.

Reply:
@hiranya911 Thanks to pointing out. Already make the change. Thank you.